### ISSUE TITLE
Issue when update Xcode 9

### ISSUE THREAD

**Issue by chandrahasan** [John-Lluch/SWRevealViewController#736]
After update  Xcode 9 , While click left side-menu button , leftside menu opened at the same time rightside menu also opened automatically , Kindly advise how to fix this issue.

![sample draw](https://user-images.githubusercontent.com/7067054/30730682-7ec39b7c-9f86-11e7-8357-38283a73d83f.png)

**Comment by iDevelopper** [comment]
I see only the right menu here!

**Comment by chandrahasan** [comment]
@iDevelopper Yes at the final we can see only RightMenu , because first left menu opened then right menu opened automatically so left menu is hidden now.

**Comment by chandrahasan** [comment]
@iDevelopper Yes at the final we can see only RightMenu , because first left menu opened then right menu opened automatically so left menu is hidden now.

**Comment by iDevelopper** [comment]
I don't think it is relative to SW, you have a mistake somewhere! How do you open the left menu?
